ng2-components/ng2-alfresco-documentlist/README.md

```diff
@@ -357,6 +360,29 @@ DocumentList provides simple breadcrumb element to indicate the current position
 | --- | --- | --- |
 | navigate | [PathElementEntity](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-core-rest-api/docs/PathElementEntity.md) |emitted when user clicks on a breadcrumb  |
 
+## Dropdown Site Component
+
+DocumentList provides simple dropdown element to indicate the current position within a navigation hierarchy.
```

Comment:
+DocumentList provides simple dropdown element to indicate the current position within a navigation hierarchy I don't think is clearing explain what is doing.